python/lsst/jointcal/jointcal.py

```diff
@@ -216,22 +230,23 @@ def run(self, ref, tract):
         chi2 = fit.ComputeChi2()
         print(chi2)
 
-        for i in range(20) :
-            r = fit.Minimize("Distortions Positions",5) # outliers removal at 5 sigma.
+        for i in range(20):
+            r = fit.Minimize("Distortions Positions", 5)  # outliers removal at 5 sigma.
```

Comment:
Thank you for the clarification. I thought "Distortions Positions" was meant to be a descriptive string; I didn't realize it was specifying what was minimized.